dev-packages/browser-integration-tests/suites/replay/captureReplay/test.ts

```diff
@@ -34,7 +34,7 @@ sentryTest('should capture replays (@sentry/browser export)', async ({ getLocalT
     event_id: expect.stringMatching(/\w{32}/),
     environment: 'production',
     sdk: {
-      integrations: [
+      integrations: expect.arrayContaining([
```

Comment:
l: any reason to loosen the assertions?

Reply:
I personally found it to be very annoying that the ordering needed to be exact. Lmk if you want me to change it back.

Reply:
ah I see, the ordering is annoying, agreed. Though these tests wouldn't fail additionally added integrations, unless I'm completely missing something. So that opens the test up to more behaviour change than previously. I don't think this is the end of the world but it'd be good to have at least one test that fails if we add a new integration by default. We should be aware of stuff like this.